iommi/style_bulma.py

```diff
@@ -19,17 +19,16 @@
 
           });
     });
-    """)
-)
+</script>
+""")
 
 bulma_base = Style(
     base,
     assets=dict(
-        css=html.link(
-            attrs__rel='stylesheet',
+        css=Asset.css(
             attrs__href='https://cdn.jsdelivr.net/npm/bulma@0.9.1/css/bulma.min.css',
         ),
-        js=_js,
+        navbar_burger_click_js__template=navbar_burger_click_js,
```

Comment:
This is an example of the kind of thing that is possible in the assets section of Part but pre this patch wasn't possible in the definition of a Style.